templates/fine-tune-llama2/README.md

```diff
@@ -0,0 +1,48 @@
+# Fine-tuning Llama-2 models with Deepspeed, Accelerate, and Ray Train
+This template shows you how to fine-tune Llama-2 models. 
+
+## Step 1: Fine-tune Llama-2-7B model
+- Open a terminal in VS Code (ctl+shift+`). 
+- Run the command below to kick off fine-tuning with dummy data, Grade School Math 8k (GSM8K) dataset.
```

Comment:
We should put a link here. Since this is meant to be available to absolute entry level, folks might not known that this is a generally known dataset and they should look at it to learn what their LLM is learning and able to do.

Reply:
which link should I use? This one? https://github.com/openai/grade-school-math

Reply:
Let's use this one: https://huggingface.co/datasets/gsm8k

Reply:
Done